cpp/cmake/thirdparty/get_cucollections.cmake

```diff
@@ -20,8 +20,8 @@ function(find_and_configure_cucollections)
     # cuCollections doesn't have a version yet
     cuco 0.0
     GLOBAL_TARGETS cuco::cuco
-    CPM_ARGS GITHUB_REPOSITORY NVIDIA/cuCollections
-    GIT_TAG f0eecb203590f1f4ac4a9f1700229f4434ac64dc
+    CPM_ARGS GITHUB_REPOSITORY PointKernel/cuCollections
+    GIT_TAG allocator-without-stream
```

Comment:
@PointKernel can you update this now that the cuco PR is merged?

Reply:
Done.